packages/matrix/helpers/index.ts

```diff
@@ -73,13 +73,8 @@ export async function reloadAndOpenAiAssistant(page: Page) {
 
 export async function openAiAssistant(page: Page) {
   await page.locator('[data-test-open-ai-assistant]').click();
-  await page.waitForFunction(() =>
-    document.querySelector('[data-test-close-ai-assistant]'),
-  );
-
-  await page.waitForFunction(() =>
-    document.querySelector('[data-test-room]')?.getAttribute('data-test-room'),
-  ); // Opening the AI assistant either opens last room or creates one - wait for it to settle
+  await expect(page.locator('[data-test-close-ai-assistant]')).toBeVisible();
```

Comment:
I think `.toHaveCount()` would be better. this is akin to `assert.dom().exists()` in Qunit. being visible has other ramifications that i don't think you want to worry about

Reply:
Updated